app/Domain/Auth/Controllers/LogoutController.php

```diff
@@ -22,6 +22,16 @@ private function getLogoutLink(): string
             Auth::guard(config('nova.guard'))->logout();
         }
 
+        if (env('AUTHENTICATION_MODE') === 'keycloak') {
+            return sprintf(
+                'https://%s/realms/%s/protocol/openid-connect/logout?client_id=%s&post_logout_redirect_uri=%s',
+                env('AUTH0_LOGIN_MANAGEMENT_DOMAIN'),
```

Comment:
Mixing `AUTH0_` with `KEYCLOAK_` in a part of the code base dedicated to Keycloak is very confusing. It is also not the best approach to streamline the cleanup once Auth0 can be removed.